cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -808,4 +810,23 @@ public String toString() {
             });
     }
 
+    public class PodRef {
+
+        private String podName;
+        private int podIndex;
```

Comment:
Can both be `final`

Reply:
And perhaps adding `namespace` allows you to simplify on of the `isReady` methods too?

Reply:
I tried adding the `namespace` but the namespace is being used directly and easily at the places where I am passing `podRef`

Reply:
OK